### ISSUE TITLE
Snap package error for CouchDB 2.1.1

### ISSUE THREAD

**Issue by fadonoso** [apache/couchdb-pkg#19]
I have a problem trying to packaging a CouchDB snap, when the source-tag is changed to 2.1.1.
The compilation fails in Fauxton. An extract of the huge error list:

```
.......

ERROR in ./app/addons/fauxton/notifications/notifications.js
ESLintError: 
/home/felipe/snap/couchdb-pkg/parts/couchdb/build/src/fauxton/app/addons/fauxton/notifications/notifications.js
   29:3   error  'getStoreState' is not defined               no-undef
   45:3   error  'onChange' is not defined                    no-undef
   49:3   error  'getStyles' is not defined                   no-undef
   72:3   error  'getNotificationCenterPanel' is not defined  no-undef
   88:3   error  'state' is not defined                       no-undef

......

```
It seems to me that the version of nodejs is not correct, although I tried with diferent version and I could not make it work (probably coz of my lack of skill with snaps).

I'm working on an ubuntu 16.04 LTS machine.
In order to reproduce this change source-tag: 2.1.0 to 2.1.1.

Does someone know how to solve this?

Thanks,
Felipe

**Comment by wohali** [comment]
The outdated 2.0.0 unsupported CouchDB snap package has been unpublished.

**Comment by gp187** [comment]
What the hell! I have multiple servers depending on the snap!

**Comment by gp187** [comment]
It was working just fine!

**Comment by wohali** [comment]
@gp187 Hi Gabriel, sorry for the inconvenience.

The reasoning is this: The package is outdated (2.0.0 vs. 2.2.0), there are multiple security issues in the released version, and we have no one to maintain it. It is irresponsible of us to publish and claim support for something that we cannot maintain ourselves. We've had multiple requests for support in IRC/Slack for this method of installation over the past few months, and they've always presented us with huge problems, for instance see apache/couchdb#1547.

You are welcome to either:
1. Use the files from this repository to build your own snap package - we didn't remove from the from the repository yet, or
1. Step up to help us address the known issues filed against this repository (#13, #17, #19, apache/couchdb#1547, apache/couchdb#1545) with PRs + upgrade the image to 2.2.0.

We'd certainly prefer the latter, and if you're willing to help us keep the snap package alive, we can absolutely re-publish it from here once the above are addressed.

Once again, sorry for the inconvenience.

**Comment by wohali** [comment]
Closed by #27.

With the new maintainer, we may republish the `couchdb` package in the snap store soon.

**Comment by user357x** [comment]
couchdb - don't relax!
